pybuildkite/agents.py

```diff
@@ -2,16 +2,26 @@
 
 
 class Agents(Client):
-
+    """
+    Agent operations for the Buildkite API
+    """
     def __init__(self, client, base_url):
         """
-
+        Construct the class
         """
         self.client = client
         self.path = base_url + 'organizations/{}/agents/'
 
     def list_all(self, organizations, name=None, hostname=None, version=None):
+        """
+        Returns a paginated list of an organization’s agents.
 
+        :param organizations: Organisation slug
```

Comment:
typo

Reply:
Oops my British spellings getting in the way!